Susi/Controllers/ForgotPasswordController/ForgotPasswordViewController.swift

```diff
@@ -26,5 +26,12 @@ class ForgotPasswordViewController: UIViewController {
         prepareResetButton()
         setupTheme()
     }
+    
+    override func localizeStrings() {
+        title = "Forgot Password".localized()
```

Comment:
Please use already declared constants from the ControllerConstants or the Constants file instead of using strings.